test/support/annotations/annotation_file.rb

```diff
@@ -0,0 +1,35 @@
+module Annotations
```

Comment:
Not that I expect us to add (but you never know what a plugin may do) a gem by this name, but there are some in existence and I could see one coming along and being used. Call it future proofing but you may consider wrapping this in `module Katello`.